torch/ao/quantization/fx/quantization_patterns.py

```diff
@@ -1486,9 +1486,16 @@ def convert(self,
                 load_arg: Callable,
                 is_reference: bool = False,
                 convert_custom_config_dict: Dict[str, Any] = None) -> Node:
-        # always produce reference pattern for relu
-        is_relu = node.op == "call_function" and node.target == torch.nn.functional.relu
-        if is_reference or is_relu:
+        # always produce reference pattern for following functions
+        func_list = [
```

Comment:
there are some missing functions here as well I think, e.g. `torch.nn.functional.relu6`, can you add all the functions supported by lowering?